models/public/yolo-v4-tf/accuracy-check.yml

```diff
@@ -29,20 +30,26 @@ models:
           - conv2d_109/BiasAdd
     datasets:
       - name: ms_coco_detection_80_class_without_background
+        im_reader: pillow
         preprocessing:
-          - type: bgr_to_rgb
-          - type: normalization
-            std: 255.0
           - type: resize
             size: 608
+            aspect_ratio_scale: fit_to_window
+            use_pillow: True
+          - type: normalization
+            std: 255.0
+          - type: padding
+            pad_type: center
         postprocessing:
-          - type: resize_prediction_boxes
+          - type: letterbox_postprocessing_resize
           - type: filter
             apply_to: prediction
             min_confidence: 0.001
             remove_filtered: true
-          - type: diou_nms
+          - type: nms
             overlap: 0.5
+            diou: True
```

Comment:
I prefer to use separated class for diou, because there are also other extensions for nms algo and they can not be used together in the same time (it will be mess).  of cause diou  can be inherited from vanilla nms

Reply:
Well, if you are stating, that nms will develop, I'll revert separation. Are you?

Reply:
As I know, there is also CIoU NMS and GIoU NMS
but do not rush to bring it back, I may change my mind :)
At the moment, I am afraid of constructions like Weighted + Cluster-DIoU-NMS (i do not look carefully in realization yet)